File: BE/src/controllers/Feedback.js
import Feedback from '../models/Feedback'

let codeMsg = {

}

exports.save = (req, res, next) => {
	let {content, contact} = req.body;
	let feedback = new Feedback;
	feedback.contact = contact;
	feedback.content = content;
	feedback.save(err => {
		if (err) return res.json({code: 10500, msg: '数据库保存失败,请重新尝试'});
		res.json({code: 10000, msg: '反馈成功'});
	})
}

File: FE/Goal/app/src/main/java/cn/goal/goal/services/Feedback.java
package cn.goal.goal.Services;

import cn.goal.goal.Utils.HttpRequest;
import org.json.JSONException;
import org.json.JSONObject;

/**
 * Created by chenlin on 18/02/2017.
 */
public class Feedback {
    /**
     * 提交反馈
     * @param content 反馈内容
     * @param contact 联系方式
     * @return 成功返回null, 错误返回错误信息
     */
    public static final String apiServer = Config.apiServer;
    public static final String feedback = Config.feedback;

    public static String submitFeedback(String content, String contact) {
        // 提交建议
        HttpRequest request = HttpRequest
                .post(apiServer + feedback)
                .form("content", content)
                .form("contact", contact);
        if (request.ok()) {
            try {
                JSONObject result = new JSONObject(request.body());
                if (result.getInt("code") == 10000) {
                    return null;
                }
                return result.getString("msg");
            } catch (JSONException e) {
                e.printStackTrace();
                return "服务器传输数据错误";
            }
        }
        return "请求失败";
    }
}